src/features/tags/components/TagManager/components/TagChip.tsx

```diff
@@ -159,6 +167,7 @@ const TagChip: React.FunctionComponent<{
         <TagToolTip tag={tag}>
           <Box className={classes.label + ' ' + classes.deleteContainer}>
             {tag.title}
+            {optionalString}
```

Comment:
Instead of this, I think we should change the `title` of the tag when rendered in this situation.